### DISCUSSION TITLE
Error Not found Telink OTA service!

### DISCUSSION THREAD

**Discussion by giuseppemari75** [pvvx/ATC_MiThermometer#570] [General]
Hi I've a one of 

![image](https://github.com/user-attachments/assets/59aeb4fe-0a6c-49f1-a687-cb39de4b206f)

but I've got the message:

![image](https://github.com/user-attachments/assets/b5999b45-a367-4835-ae3e-9a0dcb171245)

and I , sorry, don't understand if I can upgrade the firmware 

Could you please provide some support ?

Thank you so much 
Giuseppe

**Comment by pvvx** [comment]
CGG1 have more than 4 models.
The firmware is only suitable for one version, assembled on a chip from Telink. This is a variant of CGG1-M.

**Comment by giuseppemari75** [comment]
Thank you for this. Do you have any recommendations to integrate this?

**Comment by pvvx** [comment]
"MJ_HT_V1" is not "CGG1" or "CGDK2"
https://github.com/dsltip/Xiaomi_MJ_HT_V1_cc2530/tree/master/board_pics
This is a thermometer on the nRF5x chip (Nordic Semiconductor).
